benchmarks/config/base/base.json

```diff
@@ -46,6 +46,13 @@
       "flags": [ "-n", "100" ],
       "extensions": [ "(avx2|asimd)" ]
     },
+    "fp32_3x512_mlir": {
```

Comment:
What do these add over the existing 1024 case?

Reply:
removed.